exporter/honeycombexporter/honeycomb.go

```diff
@@ -177,6 +177,7 @@ func (e *honeycombExporter) pushTraceData(ctx context.Context, td consumerdata.T
 		e.sendMessageEvents(td, span, traceLevelFields)
 		e.sendSpanLinks(span)
 
+		ev.AddField("span_kind", span.Kind.String())
```

Comment:
we typically use `meta.span_type` in our taxonomy, and lowercase. see https://github.com/honeycombio/beeline-go/blob/76ad42c601cfe475c54a3c73b8ef84c2f4aabbce/trace/trace.go#L457 (cc @maplebed @paulosman)

Reply:
`meta.span_type` is currently used for `span_event` and `link` per the metadata kind docs https://docs.honeycomb.io/working-with-your-data/managing-your-data/definitions/#metadata-kind
 
If sending values other than link/span_event is alright with you, I can change it

Reply:
We need a bit of time to think through this API/schema design, is it okay with you if we come back to you in a week with a decision? Regardless, please go ahead and lowercase the constants as we're pretty sure on that.

Reply:
Yeah no rush on my end

Reply:
Separate action item: I think the exporter needs to be updated to the new internal data format, which will give it access to the other span kinds (producer, consumer, internal)